packages/web/src/pages/saved-page/components/desktop/SavedPage.tsx

```diff
@@ -39,6 +44,80 @@ const messages = {
   filterPlaceholder: 'Filter Tracks'
 }
 
+type AlbumCardProps = Pick<CardProps, 'index' | 'isLoading' | 'setDidLoad'> & {
+  album: CollectionWithOwner
+}
+
+const AlbumCard = ({ album, index, isLoading, setDidLoad }: AlbumCardProps) => {
+  const goToRoute = useGoToRoute()
+
+  const handleClick = useCallback(() => {
+    if (album.ownerHandle) {
+      goToRoute(
+        albumPage(album.ownerHandle, album.playlist_name, album.playlist_id)
+      )
+    }
+  }, [album.playlist_name, album.playlist_id, album.ownerHandle, goToRoute])
+
+  return (
+    <Card
+      index={index}
+      isLoading={isLoading}
+      setDidLoad={setDidLoad}
+      key={album.playlist_id}
+      id={album.playlist_id}
+      userId={album.playlist_owner_id}
+      imageSize={album._cover_art_sizes}
+      size='medium'
+      playlistName={album.playlist_name}
+      playlistId={album.playlist_id}
+      isPlaylist={false}
+      isPublic={!album.is_private}
+      handle={album.ownerHandle}
+      primaryText={album.playlist_name}
+      secondaryText={formatCardSecondaryText(
+        album.save_count,
+        album.playlist_contents.track_ids.length
+      )}
+      isReposted={album.has_current_user_reposted}
+      isSaved={album.has_current_user_saved}
+      cardCoverImageSizes={album._cover_art_sizes}
+      onClick={handleClick}
+    />
+  )
+}
+
+const AlbumsTabContent = () => {
+  const goToRoute = useGoToRoute()
+
+  // Temporarily requesting large page size to ensure we get all albums
+  // until the list is updated to use `InfinteScroll`
+  const { data: albums } = useSavedAlbumsDetails({ pageSize: 9999 })
+  const { isLoading, setDidLoad } = useOrderedLoad(albums.length)
+  const cards = albums.map((album, i) => {
+    return (
+      <AlbumCard
+        index={i}
+        isLoading={isLoading(i)}
+        setDidLoad={setDidLoad}
+        key={album.playlist_id}
+        album={album}
+      />
+    )
+  })
+
+  return cards.length > 0 ? (
+    <CardLineup cards={cards} cardsClassName={styles.cardsContainer} />
+  ) : (
+    <EmptyTable
+      primaryText='You haven’t favorited any albums yet.'
+      secondaryText='Once you have, this is where you’ll find them!'
+      buttonLabel='Go to Trending'
```

Comment:
nit: could pull these out into a `messages` object as campground cleaning (this file might have been around prior to that convention!)

Reply:
Oh yep. Definitely just moved code around without looking at it here. Will put into a messages object.